tools/build_config.js

```diff
@@ -44,14 +44,10 @@ module.exports = {
     },
     browser: {
       input: {
-        plugins: [
-          cjsPlugin(),
-          jsonPlugin()
-        ]
+        plugins: []
       },
       output: {
-        format: "iife",
-        outro: "return module.exports.definer || module.exports;",
+        format: "es",
```

Comment:
These changes confuse me... we still want the IIFE build... I would expect to see a `browser_esm` or perhaps just `pure_esm` section added, not the browser section altered...

Reply:
Oh I should have read it all. hmmmm..

Reply:
Yep it's a hack to avoid compiling each language module twice. I'm not too proud of it, but it works 😅 

I wasn't able to figure out what that `outro` option was doing, maybe it wasn't used after all?